sockeye/arguments.py

```diff
@@ -17,12 +17,27 @@
 import argparse
 import os
 import sys
-from typing import Callable, Optional
+from functools import partial
+from typing import Any, Callable, Dict, Tuple, Optional
 
 from sockeye.lr_scheduler import LearningRateSchedulerFixedStep
 from . import constants as C
 from . import data_io
 
+# Data structure to store argument specifications for various CLIs,
+# populated when CLI args are defined via argparse in the function below.
+FLAGS = dict()  # type: Dict[Tuple[Any], Dict[str, Any]]
```

Comment:
For me flags are a specific class of arguments. I would rename to ARGUMENTS or something similar.

Reply:
renamed